### PR TITLE
Ensure that rate and comment option is only available on approved datasets 

### PR DESCRIPTION
**Pull request by paulkurian-usc**
Ensure that rate and comment option is only available on datasets that the user has previously requested and was approved to have.

### CODE REVIEW

File: pages/artifact/review/_id.vue [hunk 1]
```diff
@@ -157,12 +174,13 @@ export default {
         } else {
         }
       }
-    }
+    },
+    async updateTicketStatus() {
+      let response = await this.$artifactRequestStatusEndpoint.show(this.artifact.artifact.artifact_group_id)
+      this.ticket_status = response.ticket_status
+      this.artifactReleased = this.ticket_status && this.ticket_status == "released"
```
Comment (hardaker) [inline pages/artifact/review/_id.vue:181]: You're missing setting artifactRequested here too right?

Comment (hardaker) [reply pages/artifact/review/_id.vue:181]: [I have a longer comment about this for a future PR but this is the only thing to fix immediately for this PR]

Comment (paulkurian-usc) [reply pages/artifact/review/_id.vue:181]: For this component, that is the rate and review component, we only care about whether the artifact is released or not. If it has been requested but unreleased we still do not allow the user to rate and review the artifact. Please let me know if my understanding of the spec is correct.

Comment (hardaker) [reply pages/artifact/review/_id.vue:181]: ah, missed the file name change in my scrolling.

**Review by hardaker — APPROVED** [review]

**Pull request merged**